roofit/roofitcore/inc/RooAddModel.h

```diff
@@ -54,12 +54,9 @@ class RooAddModel : public RooResolutionModel {
     // Return extended mode capabilities    
     return (_haveLastCoef || _allExtendable) ? MustBeExtended : CanNotBeExtended; 
   }
+  // Return expected number of events for extended likelihood calculation which
+  // is the sum of all coefficients
```

Comment:
Please use `///`, so doxygen parses it.
It doesn't make a difference here, but remember to always finish the first sentence with a `.`, so doxygen can separate the brief documentation from the rest.